devito/types/basic.py

```diff
@@ -550,12 +663,50 @@ def name(self):
     @property
     def indices(self):
         """The indices (aka dimensions) of the object."""
-        return self._indices
+        return self.args
+
+    @property
+    def index_ref(self):
```

Comment:
If we called this `indices_X` where `X` is whatever you like (base? reference? ref?), instead of `index_X`, it would be more immediate to understand the relationship/difference wr.t. `.indices`

Reply:
(offtopic: what's wr.t. ? writing/reading to?)

Reply:
with respect to

Reply:
thanks

Reply:
> If we called this indices_X where X is whatever you like (base? reference? ref?), instead of index_X, it would be more immediate to understand the relationship/difference wr.t. .indices

reminder